config/environments/development.rb

```diff
@@ -27,7 +27,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = false
+  config.assets.debug = true
```

Comment:
need this on to see the list of files in the source view ... should not be much slower since local fetch is pretty fast